charts/argo-workflows/templates/server/deployment.yaml

```diff
@@ -31,10 +29,10 @@ spec:
           httpGet:
             path: /
             port: 2746
-            {{- if .Values.server.secure }}
-            scheme: HTTPS
-            {{- else }}
+            {{- if not .Values.server.secure }}
             scheme: HTTP
+            {{- else }}
+            scheme: HTTPS
```

Comment:
These changes make the output of `./hack/compare-workflows.sh` more similar to upstream